graphics/py-gimp/Makefile

```diff
@@ -15,8 +15,11 @@ GIMP_SLAVE=	pygimp
 USES=		python:2.7
 USE_GNOME=	pygtk2
 
-CONFIGURE_ARGS=	--enable-python
-PLIST=	${.CURDIR}/pkg-plist
+BUILD_DEPENDS+=	py27-gobject3>0:devel/py-gobject3@py27 \
+		python:lang/python
```

Comment:
same here, use https://www.freebsd.org/doc/en_US.ISO8859-1/books/porters-handbook/binary-alias.html#binary-alias-ex2 instead of the metaport

Reply:
OK done.